README.md

```diff
@@ -1,14 +1,14 @@
 # GOV.UK content schemas
 
-This repo contains schemas and examples of the content formats used on GOV.UK.
+This repo contains schemas and examples of the content schemas used on GOV.UK.
```

Comment:
I don't think this makes sense any more if you replace formats with schemas. The term document type is closer in usage than format.